### ISSUE TITLE
[WPF] UserControl doesn't fill the DeskBand

### ISSUE THREAD

**Issue by kevinglier** [dsafa/CSDeskBand#51]
Hey,

I really like this project, especially its WPF-Example. When I set the MinWidth, I get DeskBand with the Size I need.

But there's a problem: The UserControl isn't correctly sized to the bounds of its parent. Is there a way to achieve this? I already tried to modify the CSDeskBand.cs, but that didn't do anything:

```
public IntPtr Handle
{
    get
    {
        if (_rootVisual.Child == null)
        {
            _rootVisual.Child = UIElement;
            _rootVisual.Child.ClipToBounds = true;
            _rootVisual.ClipToBounds = true;
        }

        return HwndSource.Handle;
    }
}
```

Do you have any other ideas?

**Comment by kevinglier** [comment]
Okay, I think I found a way by myself. The answer https://github.com/dsafa/CSDeskBand/issues/25#issuecomment-536347051 was helping me here. I think by using the overrides, it should crash at the next restart.

```
protected override void OnRenderSizeChanged(SizeChangedInfo sizeInfo)
{
    base.OnRenderSizeChanged(sizeInfo);
    SetHeightTo100Percent();
}

protected override void OnInitialized(EventArgs e)
{
    base.OnInitialized(e);
    SetHeightTo100Percent();
}

private void SetHeightTo100Percent()
{
    Height = SystemParameters.PrimaryScreenHeight - SystemParameters.WorkArea.Height;
}

```

**Comment by srwi** [comment]
In my opinion a better fix is described in #46 which fixes this for vertical and horizontal taskbars.
